gems/pending/util/postgres_admin.rb

```diff
@@ -12,6 +12,10 @@ def self.data_directory
     Pathname.new(ENV.fetch("APPLIANCE_PG_DATA"))
   end
 
+  def self.template_directory
+    Pathname.new(ENV.fetch("APPLIANCE_TEMPLATE_DIRECTORY"))
+  end
```

Comment:
I wish the system admin / postgres dba / database ops stuff was pulled out into console / appliance / or separate gem (that could be included)

maybe this value would be passed into the gem instead of using a gem. oh well. pipe dream